src/unibasicTemplates/subroutines/internal/setup.njk

```diff
@@ -7,6 +9,13 @@ S$MVOM.PROCESS = @TRUE
 
 gosub processUserDefinedOptions
 
+if udoGetProperty(setupOptions, 'maxReturnPayloadSize', maxReturnPayloadSize, type) then
```

Comment:
Is there a concern that we will not have provided a maximum size in the JS code?  It seemed like we should have some guarantees there that the maximum size will have been provided, so this check feels superfluous.

Reply:
Unfortunately it's compile error if `udoGetProperty` isn't in a conditional or doesn't assign its return value. Are you saying just throw the return value away?

Reply:
I see.  So basically you need to do something to get to the value, so it might as well fall back to a default if for some reason there's an error.  We should assume that it should never actually happen that way.  I'm okay with this.